docs/api.yaml

```diff
@@ -106,6 +106,21 @@ paths:
                             type: number
                       autoSkipSegments:
                         type: boolean
+                      autoSkipSegmentCategories:
+                        sponsor: 
+                          type: boolean
+                        intro: 
+                          type: boolean
+                        outro: 
+                          type: boolean
+                        interaction: 
+                          type: boolean
+                        selfpromo: 
+                          type: boolean
+                        music_offtopic: 
+                          type: boolean
+                        preview: 
+                          type: boolean
```

Comment:
Thanks for updating the docs, good attention to detail. Make this a reusable component to reduce the duplicated bits.